### DISCUSSION TITLE
What to set as width for iframe (with responsiveness)?

### DISCUSSION THREAD

**Discussion by fredrikaverpil** [squidfunk/mkdocs-material#5291] [Q&A]
I would like to embed some other web content on [my mkdocs-material blog](https://fredrikaverpil.github.io/blog/) using `iframe`. Something like:

```html
<iframe src="https://27b0fc68-fb63-4e94-a441-26f3f29270ac.pyscriptapps.com/91b04130-fe1c-4ec0-8a50-a3f119099153/latest/" frameborder="0" webkitallowfullscreen mozallowfullscreen allowfullscreen></iframe>
```

However, I should probably set the width, to avoid a funky look:

![image](https://user-images.githubusercontent.com/994357/228442486-94a688af-c9d8-45d3-b218-1b1e78c37cd0.png)

What's a good way to set the width, but also take responsiveness into account; different widths for different devices/resolutions?

**Comment by squidfunk** [comment]
I recommend reading [how to use a full-width iframe with custom aspect ratio](https://jameshfisher.com/2017/08/30/how-do-i-make-a-full-width-iframe/). If you don't need to maintain an aspect ratio, you can drop the surrounding divs mentioned in the article, but it's often desired.
